packages/generators/generators/lib/plops/content-type.js

```diff
@@ -1,165 +1,28 @@
-        }
-
-        return true;
-      },
-    },
-    {
-      type: 'list',
-      name: 'kind',
-      message: 'Please choose the model type',
-      default: 'collectionType',
-      choices: [
-        { name: 'Collection Type', value: 'collectionType' },
-        { name: 'Singe Type', value: 'singleType' },
-      ],
-      validate: input => validateInput(input),
-    },
-    ...getDestinationPrompts('model', plop.getDestBasePath()),
-    {
-      type: 'confirm',
-      name: 'useDraftAndPublish',
-      default: false,
-      message: 'Use draft and publish?',
-    },
-    {
-      type: 'confirm',
-      name: 'addAttributes',
-      message: 'Do you want to add attributes?',
-    },
-  ]);
-};
-
-const promptAttributeQuestions = inquirer => {
-  return inquirer.prompt([
-    {
-      type: 'input',
-      name: 'attributeName',
-      message: 'Name of attribute',
-      validate: input => validateInput(input),
-    },
-    {
-      type: 'list',
-      name: 'attributeType',
-      message: 'What type of attribute',
-      pageSize: DEFAULT_TYPES.length,
-      choices: DEFAULT_TYPES.map(type => {
-        return { name: type, value: type };
-      }),
-    },
-    {
-      when: answers => answers.attributeType === 'enumeration',
-      type: 'input',
-      name: 'enum',
-      message: 'Add values separated by a comma',
-    },
-    {
-      when: answers => answers.attributeType === 'media',
-      type: 'list',
-      name: 'multiple',
-      message: 'Choose media type',
-      choices: [{ name: 'Multiple', value: true }, { name: 'Single', value: false }],
-    },
-    {
-      type: 'confirm',
-      name: 'addAttributes',
-      message: 'Do you want to add another attribute?',
-    },
-  ]);
-};
+const ctNamesPrompts = require('./prompts/ct-names-prompts');
+const kindPrompts = require('./prompts/kind-prompts');
+const draftAndPublishPrompts = require('./prompts/draft-and-publish-prompts');
+const getAttributesPrompts = require('./prompts/get-attributes-prompts');
+const bootstrapApiPrompts = require('./prompts/bootstrap-api-prompts');
 
 module.exports = plop => {
   // Model generator
   plop.setGenerator('content-type', {
     description: 'Generate a content type for an API',
     async prompts(inquirer) {
-      const config = await promptConfigQuestions(plop, inquirer);
-
-      if (!config.addAttributes) {
-        return {
-          ...config,
-          attributes: [],
-        };
-      }
-
-      const attributes = [];
-
-      const genAttribute = async () => {
-        const answers = await promptAttributeQuestions(inquirer);
-
-        attributes.push(answers);
-
-        if (answers.addAttributes) {
-          return genAttribute();
-        }
-      };
-
-      await genAttribute();
+      const config = await inquirer.prompt([
+        ...ctNamesPrompts,
+        ...kindPrompts,
+        ...getDestinationPrompts('model', plop.getDestBasePath()),
+        ...draftAndPublishPrompts,
```

Comment:
would be good to put the API questions together. We ask where you want to put it (which api) and then is it draft & publish and this do we bootstrap api file and finally the attributes. A reorder would be great.